static/js/title-animate.js

```diff
@@ -1,50 +1,69 @@
-let title;
-let hovered = false;
+const titleStates = new Map();
 
 window.addEventListener("DOMContentLoaded", () => {
-    title = document.querySelector("nav > a > strong");
-    afterTitle = title.querySelector("::after");
-    title.addEventListener("mouseenter", showFullTitle);
-    title.addEventListener("mouseleave", hideFullTitle);
+    const titles = document.querySelectorAll(".animate");
+    titles.forEach(title => {
+        titleStates.set(title, { show: false, shown: false });
+        title.addEventListener("mouseenter", () => showFullTitle(title));
+        title.addEventListener("mouseleave", () => hideFullTitle(title));
+    });
 });
 
-function showFullTitle() {
-    setTimeout(() => {
-        title.innerText = "IMI×SHTI";
-    }, 50);
-    setTimeout(() => {
-        title.innerText = "IMI×SHOTIM";
-    }, 100);
-    setTimeout(() => {
-        title.innerText = "IMI×SHOWTIME";
-    }, 150);
-    setTimeout(() => {
-        title.innerText = "IMI×SHOWTIME_";
-        hovered = true;
-        blink(true);
-    }, 200);
+async function showFullTitle(title) {
+    const titleState = titleStates.get(title);
+    if (titleState.show) return;
+
+    titleState.show = true;
+    titleStates.set(title, titleState);
+
+    title.innerText = "SHTI";
+    await wait(50);
```

Comment:
```suggestion
    await wait(20);
```

animation sieht etwas ruckelig aus.

Reply:
Fixed